site/src/content/docs/bi-mode.mdx

```diff
@@ -0,0 +1,458 @@
+---
+title: Bi mode explanation
+description: This is a tutorial for the bi-mode using Boosted and OUDS Web.
+aliases: "/bi-mode/"
+toc: true
+---
+
+## Preamble
+
+This page is for people that want to try the bi-mode combining Boosted and OUDS Web. It’s also a tutorial to implement OUDS Web gradually inside your website. Be careful, we don’t support it too much and this feature is supposed to be used only by the selected projects. If you’re not, please contact us so we can maybe help you.
+This page is not to be used alone, but is complementary to [OUDS Web documenation](https://web.unified-design-system.orange.com/) and [Boosted documentation](https://boosted.orange.com/).
+
+## Example of integration
+
+If you want to see the kind of changes and the amount of changes it might require on your side by watching it in action in the [ODS Charts project](https://github.com/Orange-OpenSource/ods-charts/pull/673/files). Be aware that this project remains very simple due to the used technology, and that only few pages were done in this example just to test different configurations.
```

Comment:
```suggestion
You can check the [example of bi-mode on ODS Charts project](https://github.com/Orange-OpenSource/ods-charts/pull/673/files) to see what kind of changes, and how much work it would be required on your side. Be aware that the ODS Charts project remains very simple due to the used technology, and that only few pages were done in this example just to test different configurations.
```